content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -31,9 +31,53 @@ level of your CustomResourceDefinitions or advance your API to a new version wit
 
 {{< feature-state state="beta" for_kubernetes_version="1.15" >}}
 
-The CustomResourceDefinition API supports a `versions` field that you can use to
-support multiple versions of custom resources that you have developed. Versions
-can have different schemas with a conversion webhook to convert custom resources between versions.
+The CustomResourceDefinition API provides a workflow for introducing and upgrading
+to new versions of a CustomResourceDefinition.
+
+When a CustomResourceDefinition is first created, the first version is set at
+an appropriate stability level and an version number. For example `v1beta1`
+would indicate that the first version is not yet stable. All custom resource objects
+will initially be stored at this version.
+
+Once the CustomResourceDefinition is created, clients may begin using the
+`v1beta1` API.
+
+Later it might be necessary to add new version such as `v1`.
+
+When adding a version to the CustomResourceDefiition, custom resource objects
+will need to be served at both versions. This means that they will sometimes be
+served at a different version they are stored at. In order for this to be
+possible, the custom resource objects must sometimes be converted between the
+version they are stored at and the version they are served at. How custom
```

Comment:
(applies in many places) `storage version` (explained once at first use) might read more easily than `version they are stored at`. similarly, `served version` might read more easily than `version they are served at`